packages/mercury/src/tests/bundles/expected-bundle-content/utils/form.json

```diff
@@ -3,9 +3,10 @@
   ".field-block { row-gap: var(--field__row-gap); grid-template-rows: max-content 1fr; }",
   ".field-group { display: grid; }",
   ".field-group, .field-group-inline { --spacing-body-block-start: var(--mer-spacing--md); --spacing-body-block-end: var(--mer-spacing--md); --spacing-body-inline-start: var(--mer-spacing--md); --spacing-body-inline-end: var(--mer-spacing--md); row-gap: var(--field-group__row-gap); column-gap: var(--field-group__column-gap); }",
-  ".field-group-cols-2 { display: grid; column-gap: var(--field__column-gap); row-gap: var(--field-group__row-gap); grid-template-columns: max-content auto; }",
   ".field-group-inline { display: flex; flex-wrap: wrap; }",
-  ".field-inline > .label, .field-group-cols-2 .label { margin-block-start: 11px; }",
+  ".field-group-justified-end .label { text-align: end; }",
```

Comment:
Since we are updating the expected values, that means the PR title must be updated to reflect the change for the user that uses the Mercury DS implementation